### DISCUSSION TITLE
I'll cant download it!

### DISCUSSION THREAD

**Discussion by LuciferGrigo** [lllyasviel/Fooocus#3717] [Q&A]
I tryed to download with the mirrowr link but still there it stop loading! Waiting for 2 hours now but its does not work.

**Comment by p1agnut** [comment]
Hi, on this website: https://pinokio.computer/
you can find an AI browser that features among other things a Fooocus 1-click installation. Download Pinokio, install it, search for Fooocus in the Discover tab, click download. Fooocus is then downloaded and installed from this original repository with all necessary dependencies.
